### ISSUE TITLE
Two “Source string comment” confusion

### ISSUE THREAD

**Issue by Geeyun-JY3** [WeblateOrg/weblate#6870]
### Describe the problem

There are two “Source string comment” shown on translation page:
[One](https://hosted.weblate.org/translate/weblate/application/en/?checksum=f4731cb8a08a2253) is a type of String information (a section in `String information`), [the other](https://hosted.weblate.org/translate/weblate/application/en/?checksum=973561e87ae97b75) is a type of user’s comment.

IMHO, they refer to different things. The exactly same name would casue ambiguity, which may confuse users.

### Describe the solution you'd like

Avoid using `Source string comment` in the [string](https://hosted.weblate.org/translate/weblate/application/en/?checksum=973561e87ae97b75) referring to user’s comment.

### Screenshots

![image](https://user-images.githubusercontent.com/49396421/143418074-dd765f5d-18f7-4f61-9fda-40bb4d597e86.png)

### Additional context

I think some users may even confuse the `Comment` section with `Source string comment` in String information.

**Comment by nijel** [comment]
These comments have different origin:

* *Source string comment* is parsed from the translation file and is typically an explanation from developers
* *Comments* in Weblate is place to discuss. It can be used between translators of single language or at source language where it is expected to involve developers as well.

Calling the tab *Discussion* might help.

CC @orangesunny

**Comment by comradekingu** [comment]
@nijel 
Calling the "source string comment" _Discussions_ is a problem, since "discussions with other translators" is already in the "translation comment" description. It is all a bit confusing.

Source string comment, suggestions for changes to this string  https://hosted.weblate.org/translate/weblate/application/en/?checksum=973561e87ae97b75
Translation comment, **discussions** with other translators https://hosted.weblate.org/translate/weblate/application/en/?checksum=838f673cb951b3d9

I never liked the categories because they are overlapping to begin with, and since it is an extra effort to select them.

Suggested fix: One type of comments, and then a button to @ all users with "admin" in the #users of the project.

This simplifies the UI and its operation, and also consistent with the comment settings for 
[other projects](https://hosted.weblate.org/accounts/profile/#notifications__0)
[followed projects](https://hosted.weblate.org/accounts/profile/#notifications__1)
[administrated projects](https://hosted.weblate.org/accounts/profile/#notifications__2)
not differentiating between the two types of comments.

Comments are not in history, so I see no issue there (other than how they are not).

---

Mentioning users came later than the two categories of comments, and it works a lot better.
People even use @kingu more when I started adding it in the instructions. I find the comment type is not consistently selected by users, and I don't really think it has any value.(?)

If a setting for "source string type comments" is needed, then it could be "comments mentioning admins of projects".

**Comment by nijel** [comment]
Using mentions to indicate the scope will make it even more confusing for inexperienced users.

Anyway as for discussion/comments, I've made a draft pull request here: https://github.com/WeblateOrg/weblate/pull/6908, reviews welcome.

**Comment by comradekingu** [comment]
@nijel
Inexperienced and experienced users alike don't mention scope consistently now.
This means it can't be relied upon if a consideration is to be made by reading the scope and thinking it is genuine, rather than also or foremost reading the actual comment when seeing it. If the comment also has to be read, one wonders what info couldn't be inferred from it?

Rarely comments are made in languages foreign to the admin, but then those are always(?) for fellow translators.
I have trouble remembering a time when a comment was made in language foreign to me that required action on my part.
If so and it is a translation comment, you would have a true positive there I suppose.
Only that isn't the case, because translation comments are the default type of comment.
It is therefore not known whether selecting it is an active case.
It doesn't make sense to me that everyone should have to fill out the scope for this reason, and I struggle to find any other reason to do so.

If using alerts to find the comment, those don't actually distinguish between the two scopes.
So what is the use there?

To flag something as an issue it is much better to mark the source string as needing work (could be easier without changing to the language) and then making a suggestion there.
I don't do this, because marking the source string as needing work locks translations for other languages, and allowing suggestions for source languages is for some reason not the default on Hosted.
As opposed to comments, those are distinguished in alerts.

Inexperienced users use @ mentions for Twitter and possibly other things all day long.
Inexperienced users are more often using phones, and selecting drop-downs both uses valuable scree real-estate, is a more complex action, and makes the UI look more complex.
Requiring especially notice users know what a "source string" is vs. a "translation" is a cognitive load.
When everything is a "discussion" rather than a comment, I don't know what info is conveyed, but it does seem scarier.
Nobody wants a discussion (case in point here). Direct editing that the admin and others can see in digests is far superior.

TL;DR Is there an actual use for a set comment scope in any scenario?

**Comment by Geeyun-JY3** [comment]
Dont’t forget to change the document accordingly.
